pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -521,6 +532,49 @@ func (l LibvirtWrapper) SetupLibvirt() (err error) {
 	return nil
 }
 
+func getLibvirtLogsFilters() (logFilters string, enableDebugLogs bool) {
+
+	_, envVarExists := os.LookupEnv(services.ENV_VAR_LIBVIRT_DEBUG_LOGS)
+	if !envVarExists {
+		return "", false
+	}
+	launcherVerbosityLevelEnvVar, envVarExists := os.LookupEnv(services.ENV_VAR_VIRT_LAUNCHER_LOG_VERBOSITY)
+	if !envVarExists {
+		return "", false
+	}
+
+	libvirtLogVerbosity, err := strconv.Atoi(launcherVerbosityLevelEnvVar)
+	if err != nil {
+		log.Log.Infof("cannot apply LIBVIRT_DEBUG_LOGS value %s - must be a number", launcherVerbosityLevelEnvVar)
```

Comment:
The message here talks about `LIBVIRT_DEBUG_LOGS` but you're actually looking at `VIRT_LAUNCHER_LOG_VERBOSITY`.

Reply:
Done, thanks!